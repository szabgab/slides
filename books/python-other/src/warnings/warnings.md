# Warnings

* warn

{% embed include file="src/examples/warnings/warnings.py" %}



