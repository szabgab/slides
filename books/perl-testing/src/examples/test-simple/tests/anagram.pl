use strict;
use warnings;

use FindBin;
use lib "$FindBin::Bin/../lib";
use MyAnagram qw(is_anagram);

use Test::Simple tests => 5;

ok( is_anagram("abc", "abc") );
ok( is_anagram("silent", "listen") );

ok( not is_anagram("abc", "abd") );

ok( is_anagram("anagram", "nag a gram") );
ok( is_anagram("ABC", "abc") );
