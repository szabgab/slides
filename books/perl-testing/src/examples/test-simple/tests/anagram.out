1..5
ok 1
ok 2
ok 3
not ok 4
not ok 5
